src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.util.Scanner;
 public class Duke {
+    private static final int MAX_TASK_LENGTH = 100;
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
+        //Print logo and opening message
+        printLogo();
+        printGreeting();
+
+        //Receive user input and execute corresponding functions
+        getUserInput();
+
+        //Print closing message
+        printFarewell();
+    }
+
+    public static void loopEcho() {
+        Scanner sc = new Scanner(System.in);
+        String userInput = sc.nextLine();
+        while (!userInput.equals("bye")) {
+            printDivider();
+            System.out.printf("\t%s%n", userInput);
+            printDivider();
+            userInput = sc.nextLine();
+        }
+    }
+
+    public static void getUserInput() {
+        Task[] taskList = new Task[MAX_TASK_LENGTH];
+        int listIndex = 0;
+        Scanner sc = new Scanner(System.in);
+        String userInput = sc.nextLine();
+        while (!userInput.trim().toLowerCase().equals("bye")) {
```

Comment:
it may be good to not have so many chained method calls in one line as it could reduce code readability and efficiency. Perhaps an assignment could be made for userInput.trim().toLowerCase() before the loop!